source/PlayState.hx

```diff
@@ -158,31 +158,12 @@ class PlayState extends MusicBeatState
 		Conductor.mapBPMChanges(SONG);
 		Conductor.changeBPM(SONG.bpm);
 
-		switch (SONG.song.toLowerCase())
-		{
-			case 'tutorial':
-				dialogue = ["Hey you're pretty cute.", 'Use the arrow keys to keep up \nwith me singing.'];
-			case 'bopeebo':
-				dialogue = [
-					'HEY!',
-					"You think you can just sing\nwith my daughter like that?",
-					"If you want to date her...",
-					"You're going to have to go \nthrough ME first!"
-				];
-			case 'fresh':
-				dialogue = ["Not too shabby boy.", ""];
-			case 'dadbattle':
-				dialogue = [
-					"gah you think you're hot stuff?",
-					"If you can beat me here...",
-					"Only then I will even CONSIDER letting you\ndate my daughter!"
-				];
-			case 'senpai':
-				dialogue = CoolUtil.coolTextFile(Paths.txt('senpai/senpaiDialogue'));
-			case 'roses':
-				dialogue = CoolUtil.coolTextFile(Paths.txt('roses/rosesDialogue'));
-			case 'thorns':
-				dialogue = CoolUtil.coolTextFile(Paths.txt('thorns/thornsDialogue'));
+		var file:String = Paths.txt(SONG.song.toLowerCase() + '/' + SONG.song.toLowerCase() + 'Dialogue');
```

Comment:
```hx
#if sys
import sys.FileSystem;
#end
```

``` hx
function dialogueFinder():String
{
	// This may look more confusing.
	var dialogueFile:String = Paths.txt('${SONG.song.toLowerCase()}/${SONG.song.toLowerCase()}Dialogue.txt', TEXT);

	#if sys
	if (FileSystem.exists(dialogueFile))
	{
		return CoolUtil.coolTextFile(dialogueFile);
	}
	else
		return null;
	#else
	return null;
	#end
}
```

In the create function.
``` hx
dialogue = dialogueFinder();
```

Reply:
Paths already uses OpenFlAssets.

Reply:
> 
> 
> Paths already uses OpenFlAssets.

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
> 
> 
> Paths already uses OpenFlAssets.

Okay, I deleted that import